arsteinm/safety-app
# Walkthrough Name hint

## Description##
Added a hint to the walkthrough name text field so that users know what they're supposed to do because that wasn't obvious I guess.

##How to Test##
1. build and run.
2. create a new walkthrough
3. verify the hint says walkthrough name.

## Screenshot ##
![screenshot_1524528411](https://user-images.githubusercontent.com/26449216/39159123-cb8d8870-4718-11e8-93c3-1b233175f3a7.png)
